_data/banking.yml

```diff
@@ -507,10 +507,9 @@ websites:
     url: https://www.hsbc.com/
     img: hsbc.png
     tfa:
-      - software
+      - proprietary
       - hardware
-    doc: https://www.hsbc.co.uk/1/2/customer-support/online-banking-security/secure-key
-    exception: "Software 2FA requires HSBC app."
+    doc: https://www.hsbc.com/online-security
```

Comment:
I don't see anything on that page about how to enable 2FA.

Reply:
After the arrival of [PSD2](https://en.wikipedia.org/wiki/Payment_Services_Directive#Revised_Directive_on_Payment_Services_(PSD2)), you cannot really opt-in to two-factor authentication in online banking, it's enabled by default.

The old doc page only mentions the "Digital Secure Key", i.e. the software variant, so I wanted to replace that.

But you are right, this page might be better than my original proposal:
https://www.hsbc.co.uk/help/security-centre/secure-key/secure-key-help/